src/openrct2/ride/TrackDesign.cpp

```diff
@@ -1688,7 +1686,7 @@ static bool track_design_place_ride(TrackDesign* td6, int16_t x, int16_t y, int1
         x = rotatedEntranceMapPos.x + _trackPreviewOrigin.x;
         y = rotatedEntranceMapPos.y + _trackPreviewOrigin.y;
 
-        track_design_update_max_min_coordinates(x, y, z);
+        track_design_update_max_min_coordinates({ rotatedEntranceMapPos + _trackPreviewOrigin, z });
```

Comment:
I know `x` and `y` will have the same value, but if for some reason someone changes them, with this proposal your code will also work, with the diff you proposed, we might forget the coupling and break this

```suggestion
        track_design_update_max_min_coordinates({ x, y, z });
```